datahost-ld-openapi/deps.edn

```diff
@@ -52,7 +52,9 @@
 
   :test {:extra-paths ["test" "env/test/resources"]
          :extra-deps {lambdaisland/kaocha {:mvn/version "1.85.1342"}
-                      lambdaisland/kaocha-junit-xml {:mvn/version "1.17.101"}}
+                      lambdaisland/kaocha-junit-xml {:mvn/version "1.17.101"}
+											com.gfredericks/test.chuck {:mvn/version "0.2.13"}
```

Comment:
I don't think this dependency is actually used here, can we remove it for now?

Reply:
Thanks, removed!